packages/runtime/runtime-definitions/package.json

```diff
@@ -113,8 +114,7 @@
 				"forwardCompat": false,
 				"backCompat": false
 			},
-			"RemovedInterfaceDeclaration_IFluidDataStoreContextEvents": {
```

Comment:
So, it seems that `IFluidDataStoreContextEvents` had been incorrectly added here, right? It didn't change in the previous PR, so removing it is actually correct. Let's add that to the PR description if so.

Reply:
Seems totally unrelated to this change, if so I'd suggest cleaning this up in a separate PR.  How did this get out of sync anyway?

Reply:
Restore the file